Content.Server/Mech/Equipment/EntitySystems/MechGrabberSystem.cs

```diff
@@ -123,10 +123,11 @@ private void OnUiStateReady(EntityUid uid, MechGrabberComponent component, MechE
         args.States.Add(GetNetEntity(uid), state);
     }
 
-    private void OnInteract(EntityUid uid, MechGrabberComponent component, InteractNoHandEvent args)
+    private void OnInteract(EntityUid uid, MechGrabberComponent component, UserActivateInWorldEvent args)
     {
-        if (args.Handled || args.Target is not {} target)
+        if (args.Handled)
```

Comment:
Does this need complex interaction?

Reply:
no, this is just porting the old InteractNoHandEvents which act super strangely. This code in general is super cursed and i plan on rewriting it after this PR is done.